R/viz.R

```diff
@@ -1,50 +1,158 @@
 
-#' Plot a rainfall plot for one sample
+#' Plot a rainfall plot for one sample. This function takes in MAF data frame, or path to custom MAF file.
+#' If non are specified, the SSM will be obtained though GAMBLR directly.
 #'
-#' @param this_sample_id
-#' @param label_ashm_genes
-#' @param chrom
-#' @param projection
-#' @param chromosome
+#' @param this_sample_id Sample id for the sample to display. This is the only required argument.
+#' @param label_ashm_genes Boolean argument indicating whether the aSHM regions will be labeled or not.
+#' @param projection Specify projection (grch37 or hg38) of mutations. Default is grch37.
+#' @param chromosome Provide one or more chromosomes to plot. The chr prefix can be inconsistent with projection and will be handled.
+#' @param this_maf Specify custom MAF data frame of mutations.
+#' @param maf_path Specify path to MAF file if it is not already loaded into data frame.
 #'
 #'
 #' @return a ggplot2 plot. Print it using print() or save it using ggsave()
 #' @export
-#' @import ggplot2 dplyr cowplot
+#' @import ggplot2 dplyr ggrepel
 #'
 #' @examples
-prettyRainfallPlot = function(this_sample_id,label_ashm_genes=TRUE,chrom,projection="grch37",chromosome){
-  if(missing(this_sample_id)){
-    message("You must provide a sample_id to plot")
+#' prettyRainfallPlot("Raji")
```

Comment:
(What's "Raji"?)

Reply:
It is a sample id of one of the cell lines

Reply:
Right, I did assume so.